src/main/kotlin/no/nav/k9/los/nyoppgavestyring/ko/OppgaveKoTjeneste.kt

```diff
@@ -140,7 +147,7 @@ class OppgaveKoTjeneste(
         coroutineContext: CoroutineContext
     ): Pair<Oppgave, ReservasjonV3>? {
         return DetaljerMetrikker.time("taReservasjonFraKø", "hele", "$oppgaveKoId" ) {
-            doTaReservasjonFraKø(innloggetBrukerId, oppgaveKoId, coroutineContext)
+            doTaReservasjonFraKø(innloggetBrukerId, oppgaveKoId, coroutineContext).also { antallOppgaverCache.slettForKøId(oppgaveKoId) }
```

Comment:
Vil ikke denne bare undergrave 5 min refresh? Vi invaliderer jo ikke chache når det kommer nye behandlinger inn, for eksempel.